lib/typed-events.ts

```diff
@@ -26,6 +26,7 @@ export interface SerializedSocket {
   clientId: string;
   transport: string;
   nsp: string;
+  data: any;
```

Comment:
The only downside is that the `data` property will not be sent to the client every time it is updated. Would it make sense to use a `Proxy` here? Or only refresh the data when the user visits the page of socket details?